packages/anchor-utils/src/merkle/merkle-tree-factory.ts

```diff
@@ -71,9 +71,24 @@ export class MerkleTreeFactory<TData, TLeaf extends TData, TMetadata>
     const metadata = this.metadataFn ? await this.metadataFn.generateMetadata(nodes) : null
 
     const root = await this.buildLevel(nodes, 0, metadata)
+
+    const depth = this.getTreeDepth(root)
+    console.log(`Merkle tree generated with depth: ${depth}`)
```

Comment:
Can we pass in a DiagnosticsLogger instance and use that instead of raw console.log?

Reply:
1. That's temporary to verify if a config change actually changes the tree depth.
2. You'd need to change CAS code back and forth to pass the logger.

Worth it?